cdap-app-fabric/src/main/java/io/cdap/cdap/scheduler/ScheduleNotificationSubscriberService.java

```diff
@@ -199,6 +202,12 @@ protected void processNotification(ProgramScheduleStoreDataset scheduleStore,
         scheduleId = ScheduleId.fromString(scheduleIdString);
       }
 
+      if (isApplicationDisabled(scheduleId.getNamespace(), scheduleId.getApplication())) {
+        LOG.info("Application {}.{} is disabled, ignoring the schedule {}.", scheduleId.getNamespace(),
```

Comment:
Does this log happen always? If not, it should be a WARN. If it does, then it should be a DEBUG.

Reply:
Done